### ISSUE TITLE
SFNet pretrained model

### ISSUE THREAD

**Issue by purse1996** [donnyyou/torchcv#160]
I am very interested in your work on SFNet and want to test it on the cityscapes validation set. Can you provide a download link of the model?

**Comment by lxtGH** [comment]
We will release the trained model in one month in another repo for easy use. https://github.com/lxtGH/SFSegNets
